### PR TITLE
Add libpalaso l10n tasks

### PR DESCRIPTION
**Pull request by jasonleenaylor**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/libpalaso/927)

### CODE REVIEW

File: l10n/l10n.proj [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <!-- TODO
+    Build Release Palaso
+	<Exec Command="ExtractXliff -n SIL -o Palaso.dll -x Palaso.en.xlf -g ../Output/Release/SIL.*.dll" />
+	<Exec Command="overcrowdin updatefiles" />
+  -->
+  </Target>
+  <Target Name="PackageL10ns" DependsOnTargets="GetVersion">
+    <Exec Command="overcrowdin download -e -f CommonLibsL10ns.zip" />
+    <Unzip SourceFiles="CommonLibsL10ns.zip" DestinationFolder="CommonLibsL10ns" />
+    <Exec Command="&quot;$(PkgNuGet_CommandLine)\tools\NuGet.exe&quot; pack l10ns.nuspec -Version $(Version)" />
```
Comment (ermshiperete) [inline l10n/l10n.proj:24]: We might have to pass `$(GitVersion_NuGetVersion)` instead of `$(Version)`

File: l10n/l10ns.nuspec [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>libpalaso.l10ns</id>
```
Comment (ermshiperete) [inline l10n/l10ns.nuspec:4]: see above

File: l10n/l10ns.nuspec [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        <version>1.0</version>
+        <description>These are the xlif localizations of the libpalaso library.
+		It includes the localizations of all the SIL packages built from https://github.com/sillsdev/libpalaso</description>
+        <authors>Jason Naylor</authors>
+		<license type="expression">MIT</license>
+    </metadata>
```
Comment (ermshiperete) [inline l10n/l10ns.nuspec:10]: We should add two lines:

```
		<projectUrl>https://github.com/sillsdev/libpalaso</projectUrl>
		<copyright>Copyright © 2010-2020 SIL International</copyright>
```

**Comment by jasonleenaylor** [comment]
___
*[l10n/l10n.proj, line 4 at r1](https://reviewable.io/reviews/sillsdev/libpalaso/927#-M4-zErEWz83ExB8kMev-r1-4:-M4yeCIi9E3JkzOZZ84_:b-3pi1b4) ([raw file](https://github.com/sillsdev/libpalaso/blob/134e65f5ecc45b2878bb315f673ad5481c6cf644/l10n/l10n.proj#L4)):*
<details><summary><i>Previously, ermshiperete (Eberhard Beilharz) wrote…</i></summary><blockquote>

I wonder if it would be better to name the package `SIL.libpalaso.l10ns` to name it closer to the other libpalaso nuget packages - that way it would show up if you search for `SIL`
</blockquote></details>

Yes, I agree. I didn't think of that until afterward. I'll update it.

**Pull request merged**